tests/test_dataserializer.py

```diff
@@ -0,0 +1,38 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of Grid Singularity Exchange.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+from typing import Dict
+from gsy_framework.data_serializer import DataSerializer
+
+
+class TestCompressDecompressDict:
+    """Test compress and decompress dictionaries"""
+
+    @staticmethod
+    def test_compress() -> bytes:
+        """Assert if data is bytes type"""
+        data = {'key1': 'value1', 'key2': 'value2'}
+        compressed_data = DataSerializer.encode_and_compress_dict(data)
+        assert type(compressed_data) == bytes
+
+    @staticmethod
+    def test_decompress() -> Dict:
+        """Assert if data is dict type"""
+        data = DataSerializer.encode_and_compress_dict({'key1': 'value1', 'key2': 'value2'})
+        uncompressed_data = DataSerializer.decompress_and_decode(data)
+        assert type(uncompressed_data) == dict
```

Comment:
Nit: Here we're trusting that `DataSerializer.encode_and_compress_dict()` will actually do what we expect. In theory, we have no assurance about this. So we would be feeding a `data` object that we don't know much about to `DataSerializer.decompress_and_decode()`.
What we would usually try to avoid this is to pass hardcoded data. Here, the thing becomes more complicated because we're dealing with bytes objects. But in theory this could be rewritten as:
```suggestion
        data = b'x\x9ck`\x99*\xcc\x00\x01\xb5Sz\xb8\x8a\xf3sSu\xcb\x12sJS\xa7xW\x17\xeb\x01\x00p\x08\x08\xcd'
        uncompressed_data = DataSerializer.decompress_and_decode(data)
        assert uncompressed_data == {'some-value': 123}
```
To be honest, dealing with `bytes` like this is an unexplored territory for me, that's why I consider this to be a nit.